src/main/resources/application.properties

```diff
@@ -28,6 +28,9 @@ lpvs.cores=8
 lpvs.attempts=4
 
 # DB Configuration
+# The name of DB schema
+lpvs.schema=lpvs
```

Comment:
Do we need to add this to the `docker-compose.yml' file?

Reply:
In `docker-compose.yml` we use `database_dump.sql` for DB setup. The name of the schema is `lpvs`.
So, I added default values for `lpvs.schema` in the source code and we don't need this property in `docker-compose.yml`.